src/main/java/racingcar/view/InputView.java

```diff
@@ -6,20 +6,20 @@
 
     private static final String INPUT_GUIDE_MESSAGE = "경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).";
     private static final String ASK_TRY_COUNT = "시도할 횟수는 몇 회인가요?";
-    private static final Scanner scanner = new Scanner(System.in);
+    private static final Scanner SCANNER = new Scanner(System.in);
 
-    public String[] inputCarName() {
+    public String[] inputCarNames() {
         System.out.println(INPUT_GUIDE_MESSAGE);
 
-        String userInput = scanner.nextLine();
+        String userInput = SCANNER.nextLine();
 
         return userInput.split(",");
```

Comment:
아래와 같이 변수 선언 없이 바로 return 할 수 있을 것 같아요! (아래도 동일하게 변경해주세요! 😃) 
```suggestion
        return SCANNER.nextLine();
```